src/components/expression/expression-renderer.js

```diff
@@ -1023,10 +1026,13 @@ genotet.ExpressionRenderer.prototype.drawGeneProfiles_ = function() {
     this.profileHeight_ - this.profileMargins_.bottom,
     yScaleTop
   ]).domain([heatmapData.valueMin, heatmapData.valueMax]);
+  var yAxisLength = this.profileHeight_ - this.profileMargins_.bottom -
```

Comment:
This is a comment for this entire function, not for this particular line.

This function drawGeneProfiles_ is getting way too chunky and it's even very hard to read the lines, as we have many nested d3 code. I suggest refactoring this function to multiple "smaller" functions (e.g. drawProfilePaths_, drawProfilePoints_).

TODO.